% latex table generated in R 3.3.3 by xtable 1.8-2 package
% Thu Nov 30 11:47:19 2017

\documentclass[12pt,oneside,a4paper]{article}

\usepackage[T1]{fontenc}
\usepackage{longtable} 
\usepackage[latin9]{inputenc}
\usepackage{geometry, hyperref, amsmath, graphicx, sectsty}
\usepackage{wrapfig}
\usepackage{lscape}
\usepackage{rotating}
\usepackage{xcolor}
\usepackage{epstopdf}
\usepackage{setspace}
\usepackage{times}
\usepackage{cite}
\usepackage{graphicx}

\begin{document}
\fontsize{0.2cm}{0.3cm}\selectfont
\centering
\begin{longtable}{rllrrrll}

  \hline
 & GO.ID & Term & Annotated & Significant & Expected & weight01\_pval & branch \\ 
  \hline
1 & GO:0019344 & cysteine biosynthetic process & 114 &  16 & 1.98 & 1.5e-10 & BP \\ 
  2 & GO:0010027 & thylakoid membrane organization & 174 &  17 & 3.03 & 1.2e-08 & BP \\ 
  3 & GO:0015979 & photosynthesis & 396 &  51 & 6.89 & 5.6e-08 & BP \\ 
  4 & GO:0009768 & photosynthesis, light harvesting in phot... &  15 &   6 & 0.26 & 1.2e-07 & BP \\ 
  5 & GO:0010207 & photosystem II assembly & 111 &  12 & 1.93 & 5.6e-07 & BP \\ 
  6 & GO:0018298 & protein-chromophore linkage &  48 &   8 & 0.84 & 1.6e-06 & BP \\ 
  7 & GO:0019684 & photosynthesis, light reaction & 241 &  33 & 4.20 & 2.4e-06 & BP \\ 
  8 & GO:0070838 & divalent metal ion transport & 152 &  16 & 2.65 & 5.6e-06 & BP \\ 
  9 & GO:0045454 & cell redox homeostasis & 122 &  11 & 2.12 & 9.8e-06 & BP \\ 
  10 & GO:0019288 & isopentenyl diphosphate biosynthetic pro... & 178 &  13 & 3.10 & 1.6e-05 & BP \\ 
  11 & GO:0016117 & carotenoid biosynthetic process &  90 &  10 & 1.57 & 1.7e-05 & BP \\ 
  12 & GO:0006012 & galactose metabolic process &  70 &   8 & 1.22 & 2.9e-05 & BP \\ 
  13 & GO:0006364 & rRNA processing & 292 &  17 & 5.08 & 3.9e-05 & BP \\ 
  14 & GO:0015995 & chlorophyll biosynthetic process &  94 &   9 & 1.64 & 3.9e-05 & BP \\ 
  15 & GO:0006098 & pentose-phosphate shunt & 196 &  13 & 3.41 & 4.3e-05 & BP \\ 
  16 & GO:0035304 & regulation of protein dephosphorylation &  82 &   8 & 1.43 & 9.2e-05 & BP \\ 
  17 & GO:0006457 & protein folding & 275 &  15 & 4.79 & 9.6e-05 & BP \\ 
  18 & GO:0030003 & cellular cation homeostasis & 139 &  10 & 2.42 & 0.00017 & BP \\ 
  19 & GO:0032880 & regulation of protein localization &  18 &   4 & 0.31 & 0.00023 & BP \\ 
  20 & GO:0071435 & potassium ion export &   2 &   2 & 0.03 & 0.00030 & BP \\ 
  21 & GO:0045490 & pectin catabolic process &  35 &   5 & 0.61 & 0.00033 & BP \\ 
  22 & GO:0009269 & response to desiccation &   9 &   3 & 0.16 & 0.00041 & BP \\ 
  23 & GO:0000103 & sulfate assimilation &  40 &   6 & 0.70 & 0.00043 & BP \\ 
  24 & GO:0010411 & xyloglucan metabolic process &  21 &   4 & 0.37 & 0.00043 & BP \\ 
  25 & GO:0009773 & photosynthetic electron transport in pho... &  23 &   4 & 0.40 & 0.00062 & BP \\ 
  26 & GO:0009416 & response to light stimulus & 474 &  26 & 8.25 & 0.00068 & BP \\ 
  27 & GO:0006833 & water transport &  62 &   6 & 1.08 & 0.00073 & BP \\ 
  28 & GO:0007030 & Golgi organization &  86 &   7 & 1.50 & 0.00076 & BP \\ 
  29 & GO:0070588 & calcium ion transmembrane transport &  25 &   4 & 0.44 & 0.00086 & BP \\ 
  30 & GO:0019685 & photosynthesis, dark reaction &  13 &   4 & 0.23 & 0.00089 & BP \\ 
  31 & GO:0008361 & regulation of cell size &  26 &   4 & 0.45 & 0.00100 & BP \\ 
  32 & GO:0048481 & plant ovule development &  67 &   6 & 1.17 & 0.00110 & BP \\ 
  33 & GO:0009765 & photosynthesis, light harvesting &  34 &  10 & 0.59 & 0.00171 & BP \\ 
  34 & GO:0006857 & oligopeptide transport &  30 &   4 & 0.52 & 0.00174 & BP \\ 
  35 & GO:0005985 & sucrose metabolic process & 324 &  14 & 5.64 & 0.00177 & BP \\ 
  36 & GO:0010363 & regulation of plant-type hypersensitive ... &  76 &   7 & 1.32 & 0.00182 & BP \\ 
  37 & GO:0010103 & stomatal complex morphogenesis &  77 &   6 & 1.34 & 0.00225 & BP \\ 
  38 & GO:0019761 & glucosinolate biosynthetic process &  77 &   6 & 1.34 & 0.00225 & BP \\ 
  39 & GO:0006662 & glycerol ether metabolic process &  33 &   4 & 0.57 & 0.00249 & BP \\ 
  40 & GO:0045893 & positive regulation of transcription, DN... & 266 &  12 & 4.63 & 0.00259 & BP \\ 
  41 & GO:0051603 & proteolysis involved in cellular protein... & 353 &  12 & 6.15 & 0.00262 & BP \\ 
  42 & GO:0034599 & cellular response to oxidative stress &  55 &   5 & 0.96 & 0.00266 & BP \\ 
  43 & GO:0009963 & positive regulation of flavonoid biosynt... &  17 &   3 & 0.30 & 0.00297 & BP \\ 
  44 & GO:0009657 & plastid organization & 361 &  27 & 6.29 & 0.00355 & BP \\ 
  45 & GO:0006687 & glycosphingolipid metabolic process &  60 &   5 & 1.04 & 0.00389 & BP \\ 
  46 & GO:0005982 & starch metabolic process & 440 &  17 & 7.66 & 0.00393 & BP \\ 
  47 & GO:0016132 & brassinosteroid biosynthetic process &  38 &   4 & 0.66 & 0.00419 & BP \\ 
  48 & GO:0000054 & ribosomal subunit export from nucleus &   6 &   2 & 0.10 & 0.00433 & BP \\ 
  49 & GO:0006612 & protein targeting to membrane &  97 &   7 & 1.69 & 0.00486 & BP \\ 
  50 & GO:0009902 & chloroplast relocation &  92 &   6 & 1.60 & 0.00546 & BP \\ 
  51 & GO:0016126 & sterol biosynthetic process &  65 &   5 & 1.13 & 0.00549 & BP \\ 
  52 & GO:0030154 & cell differentiation & 368 &  17 & 6.41 & 0.00593 & BP \\ 
  53 & GO:0010105 & negative regulation of ethylene-activate... &   7 &   2 & 0.12 & 0.00599 & BP \\ 
  54 & GO:0006782 & protoporphyrinogen IX biosynthetic proce... &   7 &   2 & 0.12 & 0.00599 & BP \\ 
  55 & GO:0015996 & chlorophyll catabolic process &  42 &   4 & 0.73 & 0.00602 & BP \\ 
  56 & GO:0009873 & ethylene-activated signaling pathway &  39 &   5 & 0.68 & 0.00625 & BP \\ 
  57 & GO:0006027 & glycosaminoglycan catabolic process &  43 &   4 & 0.75 & 0.00655 & BP \\ 
  58 & GO:0009637 & response to blue light &  69 &   5 & 1.20 & 0.00706 & BP \\ 
  59 & GO:0061615 & glycolytic process through fructose-6-ph... &   8 &   2 & 0.14 & 0.00790 & BP \\ 
  60 & GO:0098869 & cellular oxidant detoxification & 133 &   7 & 2.32 & 0.00883 & BP \\ 
  61 & GO:0006636 & unsaturated fatty acid biosynthetic proc... &  48 &   4 & 0.84 & 0.00965 & BP \\ 
  62 & GO:0010091 & trichome branching &   9 &   2 & 0.16 & 0.01004 & BP \\ 
  63 & GO:0009965 & leaf morphogenesis & 105 &   6 & 1.83 & 0.01021 & BP \\ 
  64 & GO:0006094 & gluconeogenesis & 245 &  10 & 4.27 & 0.01114 & BP \\ 
  65 & GO:0009658 & chloroplast organization & 211 &  13 & 3.67 & 0.01150 & BP \\ 
  66 & GO:0055114 & oxidation-reduction process & 2564 &  65 & 44.64 & 0.01160 & BP \\ 
  67 & GO:0016226 & iron-sulfur cluster assembly & 108 &   6 & 1.88 & 0.01163 & BP \\ 
  68 & GO:0019253 & reductive pentose-phosphate cycle &  10 &   2 & 0.17 & 0.01241 & BP \\ 
  69 & GO:0005992 & trehalose biosynthetic process &  28 &   3 & 0.49 & 0.01244 & BP \\ 
  70 & GO:0006412 & translation & 897 &  25 & 15.62 & 0.01403 & BP \\ 
  71 & GO:0006081 & cellular aldehyde metabolic process & 418 &  23 & 7.28 & 0.01411 & BP \\ 
  72 & GO:0046487 & glyoxylate metabolic process &  73 &   6 & 1.27 & 0.01428 & BP \\ 
  73 & GO:0006749 & glutathione metabolic process &  82 &   5 & 1.43 & 0.01428 & BP \\ 
  74 & GO:0006529 & asparagine biosynthetic process &  13 &   3 & 0.23 & 0.01493 & BP \\ 
  75 & GO:0009690 & cytokinin metabolic process &  26 &   3 & 0.45 & 0.01495 & BP \\ 
  76 & GO:0000463 & maturation of LSU-rRNA from tricistronic... &  11 &   2 & 0.19 & 0.01499 & BP \\ 
  77 & GO:0016925 & protein sumoylation &  11 &   2 & 0.19 & 0.01499 & BP \\ 
  78 & GO:0048588 & developmental cell growth & 109 &   5 & 1.90 & 0.01733 & BP \\ 
  79 & GO:0060267 & positive regulation of respiratory burst &   1 &   1 & 0.02 & 0.01741 & BP \\ 
  80 & GO:0090143 & nucleoid organization &   1 &   1 & 0.02 & 0.01741 & BP \\ 
  81 & GO:0080136 & priming of cellular response to stress &   1 &   1 & 0.02 & 0.01741 & BP \\ 
  82 & GO:0000169 & activation of MAPK activity involved in ... &   1 &   1 & 0.02 & 0.01741 & BP \\ 
  83 & GO:0007263 & nitric oxide mediated signal transductio... &   1 &   1 & 0.02 & 0.01741 & BP \\ 
  84 & GO:2000037 & regulation of stomatal complex patternin... &   1 &   1 & 0.02 & 0.01741 & BP \\ 
  85 & GO:0051245 & negative regulation of cellular defense ... &   1 &   1 & 0.02 & 0.01741 & BP \\ 
  86 & GO:0034214 & protein hexamerization &   1 &   1 & 0.02 & 0.01741 & BP \\ 
  87 & GO:0009780 & photosynthetic NADP+ reduction &   1 &   1 & 0.02 & 0.01741 & BP \\ 
  88 & GO:0042493 & response to drug &  79 &   2 & 1.38 & 0.01743 & BP \\ 
  89 & GO:0006273 & lagging strand elongation &  12 &   2 & 0.21 & 0.01779 & BP \\ 
  90 & GO:0042254 & ribosome biogenesis & 745 &  33 & 12.97 & 0.01869 & BP \\ 
  91 & GO:0071555 & cell wall organization & 372 &  14 & 6.48 & 0.01954 & BP \\ 
  92 & GO:0009651 & response to salt stress & 192 &   8 & 3.34 & 0.01968 & BP \\ 
  93 & GO:0015994 & chlorophyll metabolic process & 196 &  16 & 3.41 & 0.02022 & BP \\ 
  94 & GO:0009926 & auxin polar transport &  61 &   4 & 1.06 & 0.02171 & BP \\ 
  95 & GO:0001731 & formation of translation preinitiation c... &  14 &   2 & 0.24 & 0.02397 & BP \\ 
  96 & GO:0051103 & DNA ligation involved in DNA repair &  14 &   2 & 0.24 & 0.02397 & BP \\ 
  97 & GO:0010206 & photosystem II repair &  15 &   2 & 0.26 & 0.02734 & BP \\ 
  98 & GO:0006002 & fructose 6-phosphate metabolic process &  15 &   2 & 0.26 & 0.02734 & BP \\ 
  99 & GO:0010015 & root morphogenesis & 145 &   7 & 2.52 & 0.02980 & BP \\ 
  100 & GO:0042793 & transcription from plastid promoter &  68 &   4 & 1.18 & 0.03085 & BP \\ 
  101 & GO:0016168 & chlorophyll binding &  40 &  10 & 0.66 & 7.7e-10 & MF \\ 
  102 & GO:0031409 & pigment binding &  14 &   5 & 0.23 & 2.2e-06 & MF \\ 
  103 & GO:0030570 & pectate lyase activity &   9 &   4 & 0.15 & 8.8e-06 & MF \\ 
  104 & GO:0005515 & protein binding & 3192 &  88 & 52.91 & 2.3e-05 & MF \\ 
  105 & GO:0016762 & xyloglucan:xyloglucosyl transferase acti... &  16 &   4 & 0.27 & 0.00012 & MF \\ 
  106 & GO:0015035 & protein disulfide oxidoreductase activit... &  77 &   7 & 1.28 & 0.00029 & MF \\ 
  107 & GO:0042803 & protein homodimerization activity &  62 &   6 & 1.03 & 0.00056 & MF \\ 
  108 & GO:0004565 & beta-galactosidase activity &  24 &   4 & 0.40 & 0.00061 & MF \\ 
  109 & GO:0003825 & alpha,alpha-trehalose-phosphate synthase... &  11 &   3 & 0.18 & 0.00068 & MF \\ 
  110 & GO:0051740 & ethylene binding &   3 &   2 & 0.05 & 0.00081 & MF \\ 
  111 & GO:0016630 & protochlorophyllide reductase activity &   3 &   2 & 0.05 & 0.00081 & MF \\ 
  112 & GO:0004066 & asparagine synthase (glutamine-hydrolyzi... &  13 &   3 & 0.22 & 0.00114 & MF \\ 
  113 & GO:0016874 & ligase activity & 726 &  23 & 12.03 & 0.00135 & MF \\ 
  114 & GO:0016671 & oxidoreductase activity, acting on a sul... &  54 &   6 & 0.90 & 0.00152 & MF \\ 
  115 & GO:0003950 & NAD+ ADP-ribosyltransferase activity &  32 &   4 & 0.53 & 0.00186 & MF \\ 
  116 & GO:0003735 & structural constituent of ribosome & 432 &  16 & 7.16 & 0.00252 & MF \\ 
  117 & GO:0008177 & succinate dehydrogenase (ubiquinone) act... &   5 &   2 & 0.08 & 0.00265 & MF \\ 
  118 & GO:0005509 & calcium ion binding & 320 &  13 & 5.30 & 0.00285 & MF \\ 
  119 & GO:0051082 & unfolded protein binding &  91 &   6 & 1.51 & 0.00409 & MF \\ 
  120 & GO:0004364 & glutathione transferase activity &  40 &   4 & 0.66 & 0.00425 & MF \\ 
  121 & GO:0004805 & trehalose-phosphatase activity &  21 &   3 & 0.35 & 0.00482 & MF \\ 
  122 & GO:0004497 & monooxygenase activity & 216 &  10 & 3.58 & 0.00632 & MF \\ 
  123 & GO:0046556 & alpha-L-arabinofuranosidase activity &   8 &   2 & 0.13 & 0.00719 & MF \\ 
  124 & GO:0010277 & chlorophyllide a oxygenase [overall] act... &   8 &   2 & 0.13 & 0.00719 & MF \\ 
  125 & GO:0003872 & 6-phosphofructokinase activity &  10 &   2 & 0.17 & 0.01130 & MF \\ 
  126 & GO:0016705 & oxidoreductase activity, acting on paire... & 287 &  10 & 4.76 & 0.01267 & MF \\ 
  127 & GO:0030246 & carbohydrate binding & 192 &   8 & 3.18 & 0.01514 & MF \\ 
  128 & GO:0001055 & RNA polymerase II activity &  12 &   2 & 0.20 & 0.01621 & MF \\ 
  129 & GO:0032441 & pheophorbide a oxygenase activity &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  130 & GO:0033947 & mannosylglycoprotein endo-beta-mannosida... &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  131 & GO:0080124 & pheophytinase activity &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  132 & GO:0052624 & 2-phytyl-1,4-naphthoquinone methyltransf... &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  133 & GO:0048529 & magnesium-protoporphyrin IX monomethyl e... &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  134 & GO:0004558 & alpha-1,4-glucosidase activity &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  135 & GO:0005222 & intracellular cAMP activated cation chan... &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  136 & GO:0043864 & indoleacetamide hydrolase activity &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  137 & GO:0050178 & phenylpyruvate tautomerase activity &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  138 & GO:0038199 & ethylene receptor activity &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  139 & GO:0045435 & lycopene epsilon cyclase activity &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  140 & GO:0004038 & allantoinase activity &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  141 & GO:0052893 & N1-acetylspermine:oxygen oxidoreductase ... &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  142 & GO:0052896 & spermidine oxidase (propane-1,3-diamine-... &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  143 & GO:0052897 & N8-acetylspermidine:oxygen oxidoreductas... &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  144 & GO:0032450 & maltose alpha-glucosidase activity &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  145 & GO:0052898 & N1-acetylspermidine:oxygen oxidoreductas... &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  146 & GO:0052900 & spermine oxidase (propane-1,3-diamine-fo... &   1 &   1 & 0.02 & 0.01658 & MF \\ 
  147 & GO:0005524 & ATP binding & 3193 &  68 & 52.93 & 0.01825 & MF \\ 
  148 & GO:0016984 & ribulose-bisphosphate carboxylase activi... &  14 &   2 & 0.23 & 0.02187 & MF \\ 
  149 & GO:0015254 & glycerol channel activity &  16 &   2 & 0.27 & 0.02822 & MF \\ 
  150 & GO:0046872 & metal ion binding & 5795 &  94 & 96.06 & 0.02980 & MF \\ 
  151 & GO:0015250 & water channel activity &  17 &   2 & 0.28 & 0.03164 & MF \\ 
  152 & GO:0045174 & glutathione dehydrogenase (ascorbate) ac... &   2 &   1 & 0.03 & 0.03288 & MF \\ 
  153 & GO:0080123 & jasmonate-amino synthetase activity &   2 &   1 & 0.03 & 0.03288 & MF \\ 
  154 & GO:0050203 & oxalate-CoA ligase activity &   2 &   1 & 0.03 & 0.03288 & MF \\ 
  155 & GO:0019148 & D-cysteine desulfhydrase activity &   2 &   1 & 0.03 & 0.03288 & MF \\ 
  156 & GO:0008660 & 1-aminocyclopropane-1-carboxylate deamin... &   2 &   1 & 0.03 & 0.03288 & MF \\ 
  157 & GO:0080048 & GDP-D-glucose phosphorylase activity &   2 &   1 & 0.03 & 0.03288 & MF \\ 
  158 & GO:0047995 & hydroxyphenylpyruvate reductase activity &   2 &   1 & 0.03 & 0.03288 & MF \\ 
  159 & GO:0004818 & glutamate-tRNA ligase activity &   2 &   1 & 0.03 & 0.03288 & MF \\ 
  160 & GO:0009916 & alternative oxidase activity &   2 &   1 & 0.03 & 0.03288 & MF \\ 
  161 & GO:0003954 & NADH dehydrogenase activity & 100 &   3 & 1.66 & 0.03526 & MF \\ 
  162 & GO:0003910 & DNA ligase (ATP) activity &  20 &   2 & 0.33 & 0.04279 & MF \\ 
  163 & GO:0050897 & cobalt ion binding &  47 &   3 & 0.78 & 0.04289 & MF \\ 
  164 & GO:0020037 & heme binding & 328 &  10 & 5.44 & 0.04805 & MF \\ 
  165 & GO:0004604 & phosphoadenylyl-sulfate reductase (thior... &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  166 & GO:0004758 & serine C-palmitoyltransferase activity &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  167 & GO:0042286 & glutamate-1-semialdehyde 2,1-aminomutase... &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  168 & GO:0004335 & galactokinase activity &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  169 & GO:0051538 & 3 iron, 4 sulfur cluster binding &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  170 & GO:0008878 & glucose-1-phosphate adenylyltransferase ... &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  171 & GO:0048027 & mRNA 5'-UTR binding &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  172 & GO:0005242 & inward rectifier potassium channel activ... &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  173 & GO:0001053 & plastid sigma factor activity &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  174 & GO:0004853 & uroporphyrinogen decarboxylase activity &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  175 & GO:0004418 & hydroxymethylbilane synthase activity &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  176 & GO:0004816 & asparagine-tRNA ligase activity &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  177 & GO:0004615 & phosphomannomutase activity &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  178 & GO:0008124 & 4-alpha-hydroxytetrahydrobiopterin dehyd... &   3 &   1 & 0.05 & 0.04891 & MF \\ 
  179 & GO:0051537 & 2 iron, 2 sulfur cluster binding &  51 &   3 & 0.85 & 0.05252 & MF \\ 
  180 & GO:0004674 & protein serine/threonine kinase activity & 607 &  16 & 10.06 & 0.05407 & MF \\ 
  181 & GO:0008987 & quinolinate synthetase A activity &   4 &   1 & 0.07 & 0.06468 & MF \\ 
  182 & GO:0004742 & dihydrolipoyllysine-residue acetyltransf... &   4 &   1 & 0.07 & 0.06468 & MF \\ 
  183 & GO:0005262 & calcium channel activity &   4 &   1 & 0.07 & 0.06468 & MF \\ 
  184 & GO:0003852 & 2-isopropylmalate synthase activity &   4 &   1 & 0.07 & 0.06468 & MF \\ 
  185 & GO:0004047 & aminomethyltransferase activity &   4 &   1 & 0.07 & 0.06468 & MF \\ 
  186 & GO:0004435 & phosphatidylinositol phospholipase C act... &   4 &   1 & 0.07 & 0.06468 & MF \\ 
  187 & GO:0003959 & NADPH dehydrogenase activity &   4 &   1 & 0.07 & 0.06468 & MF \\ 
  188 & GO:0004197 & cysteine-type endopeptidase activity &  32 &   3 & 0.53 & 0.06838 & MF \\ 
  189 & GO:0005544 & calcium-dependent phospholipid binding &  27 &   2 & 0.45 & 0.07332 & MF \\ 
  190 & GO:0005506 & iron ion binding & 310 &   9 & 5.14 & 0.07469 & MF \\ 
  191 & GO:0004185 & serine-type carboxypeptidase activity &  60 &   3 & 0.99 & 0.07756 & MF \\ 
  192 & GO:0046906 & tetrapyrrole binding & 373 &  21 & 6.18 & 0.07777 & MF \\ 
  193 & GO:0032442 & phenylcoumaran benzylic ether reductase ... &   5 &   1 & 0.08 & 0.08019 & MF \\ 
  194 & GO:0031386 & protein tag &   5 &   1 & 0.08 & 0.08019 & MF \\ 
  195 & GO:0050236 & pyridoxine:NADP 4-dehydrogenase activity &   5 &   1 & 0.08 & 0.08019 & MF \\ 
  196 & GO:0019789 & SUMO transferase activity &   5 &   1 & 0.08 & 0.08019 & MF \\ 
  197 & GO:0003999 & adenine phosphoribosyltransferase activi... &   5 &   1 & 0.08 & 0.08019 & MF \\ 
  198 & GO:0032440 & 2-alkenal reductase [NAD(P)] activity &  63 &   3 & 1.04 & 0.08688 & MF \\ 
  199 & GO:0003713 & transcription coactivator activity &  30 &   2 & 0.50 & 0.08801 & MF \\ 
  200 & GO:0008289 & lipid binding & 174 &   7 & 2.88 & 0.09127 & MF \\ 
  201 & GO:0009535 & chloroplast thylakoid membrane & 227 &  30 & 4.70 & 9.4e-14 & CC \\ 
  202 & GO:0010287 & plastoglobule &  24 &   8 & 0.50 & 1.7e-08 & CC \\ 
  203 & GO:0009941 & chloroplast envelope & 425 &  29 & 8.80 & 2.0e-08 & CC \\ 
  204 & GO:0048046 & apoplast & 174 &  18 & 3.60 & 2.3e-08 & CC \\ 
  205 & GO:0009522 & photosystem I &  54 &  15 & 1.12 & 3.1e-07 & CC \\ 
  206 & GO:0009538 & photosystem I reaction center &   5 &   4 & 0.10 & 8.9e-07 & CC \\ 
  207 & GO:0009534 & chloroplast thylakoid & 310 &  40 & 6.42 & 4.5e-06 & CC \\ 
  208 & GO:0005618 & cell wall & 295 &  18 & 6.11 & 1.6e-05 & CC \\ 
  209 & GO:0009507 & chloroplast & 1878 &  96 & 38.87 & 0.00015 & CC \\ 
  210 & GO:0031977 & thylakoid lumen &  45 &   6 & 0.93 & 0.00031 & CC \\ 
  211 & GO:0009523 & photosystem II & 129 &  14 & 2.67 & 0.00055 & CC \\ 
  212 & GO:0009570 & chloroplast stroma & 491 &  31 & 10.16 & 0.00094 & CC \\ 
  213 & GO:0009341 & beta-galactosidase complex &  23 &   4 & 0.48 & 0.00117 & CC \\ 
  214 & GO:0030076 & light-harvesting complex &   6 &   3 & 0.12 & 0.00126 & CC \\ 
  215 & GO:0030093 & chloroplast photosystem I &   3 &   2 & 0.06 & 0.00126 & CC \\ 
  216 & GO:0000795 & synaptonemal complex &  14 &   3 & 0.29 & 0.00270 & CC \\ 
  217 & GO:0009654 & photosystem II oxygen evolving complex &  50 &   5 & 1.04 & 0.00365 & CC \\ 
  218 & GO:0005622 & intracellular & 12244 & 308 & 253.45 & 0.00407 & CC \\ 
  219 & GO:0019898 & extrinsic component of membrane &  96 &   7 & 1.99 & 0.00702 & CC \\ 
  220 & GO:0009573 & chloroplast ribulose bisphosphate carbox... &   7 &   2 & 0.14 & 0.00838 & CC \\ 
  221 & GO:0005763 & mitochondrial small ribosomal subunit &   9 &   2 & 0.19 & 0.01398 & CC \\ 
  222 & GO:0031410 & cytoplasmic vesicle & 802 &  26 & 16.60 & 0.01644 & CC \\ 
  223 & GO:0017177 & glucosidase II complex &  10 &   2 & 0.21 & 0.01723 & CC \\ 
  224 & GO:0005945 & 6-phosphofructokinase complex &  10 &   2 & 0.21 & 0.01723 & CC \\ 
  225 & GO:0005794 & Golgi apparatus & 526 &  14 & 10.89 & 0.01986 & CC \\ 
  226 & GO:0005623 & cell & 13500 & 325 & 279.45 & 0.02183 & CC \\ 
  227 & GO:0009506 & plasmodesma & 338 &  13 & 7.00 & 0.02442 & CC \\ 
  228 & GO:0016282 & eukaryotic 43S preinitiation complex &  13 &   2 & 0.27 & 0.02867 & CC \\ 
  229 & GO:0005852 & eukaryotic translation initiation factor... &  14 &   2 & 0.29 & 0.03300 & CC \\ 
  230 & GO:0005737 & cytoplasm & 8378 & 236 & 173.43 & 0.03650 & CC \\ 
  231 & GO:0033290 & eukaryotic 48S preinitiation complex &  15 &   2 & 0.31 & 0.03757 & CC \\ 
  232 & GO:0034357 & photosynthetic membrane & 347 &  40 & 7.18 & 0.03764 & CC \\ 
  233 & GO:0009783 & photosystem II antenna complex &   2 &   1 & 0.04 & 0.04097 & CC \\ 
  234 & GO:0009332 & glutamate-tRNA ligase complex &   2 &   1 & 0.04 & 0.04097 & CC \\ 
  235 & GO:0080085 & signal recognition particle, chloroplast... &   2 &   1 & 0.04 & 0.04097 & CC \\ 
  236 & GO:0005764 & lysosome &  17 &   2 & 0.35 & 0.04736 & CC \\ 
  237 & GO:0005840 & ribosome & 636 &  23 & 13.17 & 0.04811 & CC \\ 
  238 & GO:0005665 & DNA-directed RNA polymerase II, core com... &  18 &   2 & 0.37 & 0.05257 & CC \\ 
  239 & GO:0045281 & succinate dehydrogenase complex &   8 &   2 & 0.17 & 0.06069 & CC \\ 
  240 & GO:0000311 & plastid large ribosomal subunit &   3 &   1 & 0.06 & 0.06083 & CC \\ 
  241 & GO:0070545 & PeBoW complex &   3 &   1 & 0.06 & 0.06083 & CC \\ 
  242 & GO:0017059 & serine C-palmitoyltransferase complex &   3 &   1 & 0.06 & 0.06083 & CC \\ 
  243 & GO:0031519 & PcG protein complex &   3 &   1 & 0.06 & 0.06083 & CC \\ 
  244 & GO:0017071 & intracellular cyclic nucleotide activate... &   3 &   1 & 0.06 & 0.06083 & CC \\ 
  245 & GO:0009517 & PSII associated light-harvesting complex... &   3 &   1 & 0.06 & 0.06083 & CC \\ 
  246 & GO:0009579 & thylakoid & 490 &  53 & 10.14 & 0.07150 & CC \\ 
  247 & GO:0042765 & GPI-anchor transamidase complex &   4 &   1 & 0.08 & 0.08027 & CC \\ 
  248 & GO:0022626 & cytosolic ribosome & 158 &   6 & 3.27 & 0.09337 & CC \\ 
  249 & GO:0044434 & chloroplast part & 970 &  63 & 20.08 & 0.09916 & CC \\ 
  250 & GO:0030688 & preribosome, small subunit precursor &   5 &   1 & 0.10 & 0.09931 & CC \\ 
  251 & GO:0010007 & magnesium chelatase complex &   5 &   1 & 0.10 & 0.09931 & CC \\ 
  252 & GO:0005749 & mitochondrial respiratory chain complex ... &   5 &   1 & 0.10 & 0.09931 & CC \\ 
  253 & GO:0043073 & germ cell nucleus &   5 &   1 & 0.10 & 0.09931 & CC \\ 
  254 & GO:0005948 & acetolactate synthase complex &   5 &   1 & 0.10 & 0.09931 & CC \\ 
  255 & GO:0031201 & SNARE complex &  28 &   2 & 0.58 & 0.11367 & CC \\ 
  256 & GO:0005787 & signal peptidase complex &   6 &   1 & 0.12 & 0.11796 & CC \\ 
  257 & GO:0005829 & cytosol & 1091 &  31 & 22.58 & 0.12385 & CC \\ 
  258 & GO:0042651 & thylakoid membrane & 276 &  35 & 5.71 & 0.12685 & CC \\ 
  259 & GO:0009325 & nitrate reductase complex &   7 &   1 & 0.14 & 0.13623 & CC \\ 
  260 & GO:0042644 & chloroplast nucleoid &   7 &   1 & 0.14 & 0.13623 & CC \\ 
  261 & GO:0005744 & mitochondrial inner membrane presequence... &   7 &   1 & 0.14 & 0.13623 & CC \\ 
  262 & GO:0009539 & photosystem II reaction center &   7 &   1 & 0.14 & 0.13623 & CC \\ 
  263 & GO:0031012 & extracellular matrix &   7 &   1 & 0.14 & 0.13623 & CC \\ 
  264 & GO:0005615 & extracellular space &  33 &   2 & 0.68 & 0.14868 & CC \\ 
  265 & GO:0005742 & mitochondrial outer membrane translocase... &   8 &   1 & 0.17 & 0.15411 & CC \\ 
  266 & GO:0045254 & pyruvate dehydrogenase complex &   8 &   1 & 0.17 & 0.15411 & CC \\ 
  267 & GO:0010598 & NAD(P)H dehydrogenase complex (plastoqui... &   8 &   1 & 0.17 & 0.15411 & CC \\ 
  268 & GO:0019773 & proteasome core complex, alpha-subunit c... &   8 &   1 & 0.17 & 0.15411 & CC \\ 
  269 & GO:0070847 & core mediator complex &   9 &   1 & 0.19 & 0.17163 & CC \\ 
  270 & GO:0030687 & preribosome, large subunit precursor &   9 &   1 & 0.19 & 0.17163 & CC \\ 
  271 & GO:0005773 & vacuole & 522 &  16 & 10.81 & 0.17497 & CC \\ 
  272 & GO:0022627 & cytosolic small ribosomal subunit &  37 &   2 & 0.77 & 0.17803 & CC \\ 
  273 & GO:0009508 & plastid chromosome &  10 &   1 & 0.21 & 0.18879 & CC \\ 
  274 & GO:0016272 & prefoldin complex &  10 &   1 & 0.21 & 0.18879 & CC \\ 
  275 & GO:0031969 & chloroplast membrane &  75 &   3 & 1.55 & 0.20300 & CC \\ 
  276 & GO:0016529 & sarcoplasmic reticulum &  11 &   1 & 0.23 & 0.20559 & CC \\ 
  277 & GO:0005886 & plasma membrane & 1336 &  27 & 27.66 & 0.20911 & CC \\ 
  278 & GO:0030176 & integral component of endoplasmic reticu... &  16 &   2 & 0.33 & 0.22158 & CC \\ 
  279 & GO:0005736 & DNA-directed RNA polymerase I complex &  13 &   1 & 0.27 & 0.23816 & CC \\ 
  280 & GO:0005834 & heterotrimeric G-protein complex &  13 &   1 & 0.27 & 0.23816 & CC \\ 
  281 & GO:0005743 & mitochondrial inner membrane & 178 &   6 & 3.68 & 0.24538 & CC \\ 
  282 & GO:0042720 & mitochondrial inner membrane peptidase c... &  14 &   1 & 0.29 & 0.25394 & CC \\ 
  283 & GO:0000123 & histone acetyltransferase complex &  49 &   2 & 1.01 & 0.26954 & CC \\ 
  284 & GO:0031248 & protein acetyltransferase complex &  65 &   3 & 1.35 & 0.28392 & CC \\ 
  285 & GO:0016592 & mediator complex &  54 &   2 & 1.12 & 0.30798 & CC \\ 
  286 & GO:0005654 & nucleoplasm & 257 &   7 & 5.32 & 0.31316 & CC \\ 
  287 & GO:0019013 & viral nucleocapsid &  55 &   2 & 1.14 & 0.31562 & CC \\ 
  288 & GO:0009505 & plant-type cell wall &  97 &   3 & 2.01 & 0.32547 & CC \\ 
  289 & GO:0070469 & respiratory chain & 123 &   3 & 2.55 & 0.32819 & CC \\ 
  290 & GO:0005666 & DNA-directed RNA polymerase III complex &  21 &   1 & 0.43 & 0.35564 & CC \\ 
  291 & GO:0005667 & transcription factor complex & 785 &  18 & 16.25 & 0.35835 & CC \\ 
  292 & GO:0009536 & plastid & 2777 & 113 & 57.48 & 0.37357 & CC \\ 
  293 & GO:0005774 & vacuolar membrane & 239 &   6 & 4.95 & 0.37461 & CC \\ 
  294 & GO:0031224 & intrinsic component of membrane & 6538 & 121 & 135.34 & 0.38567 & CC \\ 
  295 & GO:0045261 & proton-transporting ATP synthase complex... &  24 &   1 & 0.50 & 0.39488 & CC \\ 
  296 & GO:0000775 & chromosome, centromeric region &  25 &   1 & 0.52 & 0.40742 & CC \\ 
  297 & GO:0010319 & stromule &  25 &   1 & 0.52 & 0.40742 & CC \\ 
  298 & GO:0005762 & mitochondrial large ribosomal subunit &  29 &   1 & 0.60 & 0.45505 & CC \\ 
  299 & GO:0000136 & alpha-1,6-mannosyltransferase complex &  31 &   1 & 0.64 & 0.47741 & CC \\ 
  300 & GO:0005856 & cytoskeleton & 269 &   4 & 5.57 & 0.47910 & CC \\ 
  \hline
%\caption{A \code{longtable} spanning several pages} 
\end{longtable}


% latex table generated in R 3.4.3 by xtable 1.8-2 package
% Sun Dec 10 19:42:26 2017

\begin{longtable}{rllrrrll}
  \hline
 & GO.ID & Term & Annotated & Significant & Expected & weight01\_pval & branch \\ 
  \hline
1 & GO:0019288 & isopentenyl diphosphate biosynthetic pro... & 178 & 109 & 22.72 & $<$ 1e-30 & BP \\ 
  2 & GO:0006098 & pentose-phosphate shunt & 196 & 103 & 25.02 & $<$ 1e-30 & BP \\ 
  3 & GO:0042254 & ribosome biogenesis & 745 & 270 & 95.09 & $<$ 1e-30 & BP \\ 
  4 & GO:0006412 & translation & 897 & 285 & 114.50 & $<$ 1e-30 & BP \\ 
  5 & GO:0010027 & thylakoid membrane organization & 174 &  84 & 22.21 & 2.9e-30 & BP \\ 
  6 & GO:0010207 & photosystem II assembly & 111 &  62 & 14.17 & 3.2e-27 & BP \\ 
  7 & GO:0006364 & rRNA processing & 292 & 106 & 37.27 & 6.2e-27 & BP \\ 
  8 & GO:0016117 & carotenoid biosynthetic process &  90 &  56 & 11.49 & 1.1e-25 & BP \\ 
  9 & GO:0019252 & starch biosynthetic process & 122 &  63 & 15.57 & 9.6e-25 & BP \\ 
  10 & GO:0000023 & maltose metabolic process & 105 &  55 & 13.40 & 8.9e-22 & BP \\ 
  11 & GO:0015995 & chlorophyll biosynthetic process &  94 &  50 & 12.00 & 1.0e-21 & BP \\ 
  12 & GO:0043085 & positive regulation of catalytic activit... & 253 &  72 & 32.29 & 3.1e-21 & BP \\ 
  13 & GO:0019344 & cysteine biosynthetic process & 114 &  53 & 14.55 & 5.5e-20 & BP \\ 
  14 & GO:0009902 & chloroplast relocation &  92 &  50 & 11.74 & 1.8e-19 & BP \\ 
  15 & GO:0051788 & response to misfolded protein &  80 &  43 & 10.21 & 1.9e-18 & BP \\ 
  16 & GO:0006457 & protein folding & 275 &  98 & 35.10 & 4.2e-17 & BP \\ 
  17 & GO:0006096 & glycolytic process & 228 &  80 & 29.10 & 6.2e-17 & BP \\ 
  18 & GO:0035304 & regulation of protein dephosphorylation &  82 &  39 & 10.47 & 1.7e-16 & BP \\ 
  19 & GO:0009637 & response to blue light &  69 &  38 & 8.81 & 3.8e-16 & BP \\ 
  20 & GO:0080129 & proteasome core complex assembly &  60 &  34 & 7.66 & 8.0e-16 & BP \\ 
  21 & GO:0006636 & unsaturated fatty acid biosynthetic proc... &  48 &  30 & 6.13 & 9.4e-16 & BP \\ 
  22 & GO:0006094 & gluconeogenesis & 245 &  79 & 31.27 & 1.4e-15 & BP \\ 
  23 & GO:0010114 & response to red light &  43 &  28 & 5.49 & 1.8e-15 & BP \\ 
  24 & GO:0006511 & ubiquitin-dependent protein catabolic pr... & 262 & 102 & 33.44 & 2.4e-15 & BP \\ 
  25 & GO:0009853 & photorespiration &  74 &  38 & 9.45 & 5.4e-15 & BP \\ 
  26 & GO:0010155 & regulation of proton transport &  34 &  24 & 4.34 & 1.2e-14 & BP \\ 
  27 & GO:0043161 & proteasome-mediated ubiquitin-dependent ... & 109 &  46 & 13.91 & 1.6e-14 & BP \\ 
  28 & GO:0045893 & positive regulation of transcription, DN... & 266 &  81 & 33.95 & 2.0e-13 & BP \\ 
  29 & GO:0015976 & carbon utilization & 102 &  44 & 13.02 & 2.0e-13 & BP \\ 
  30 & GO:0007264 & small GTPase mediated signal transductio... & 112 &  40 & 14.30 & 9.6e-13 & BP \\ 
  31 & GO:0009651 & response to salt stress & 192 &  62 & 24.51 & 1.3e-12 & BP \\ 
  32 & GO:0009773 & photosynthetic electron transport in pho... &  23 &  18 & 2.94 & 1.4e-12 & BP \\ 
  33 & GO:0019761 & glucosinolate biosynthetic process &  77 &  35 & 9.83 & 1.8e-12 & BP \\ 
  34 & GO:0019684 & photosynthesis, light reaction & 241 & 130 & 30.76 & 2.0e-12 & BP \\ 
  35 & GO:0010218 & response to far red light &  47 &  26 & 6.00 & 4.3e-12 & BP \\ 
  36 & GO:0006446 & regulation of translational initiation &  91 &  38 & 11.62 & 5.1e-12 & BP \\ 
  37 & GO:0015979 & photosynthesis & 396 & 184 & 50.55 & 2.9e-11 & BP \\ 
  38 & GO:0006119 & oxidative phosphorylation & 203 &  71 & 25.91 & 5.7e-11 & BP \\ 
  39 & GO:0030154 & cell differentiation & 368 &  87 & 46.97 & 5.9e-11 & BP \\ 
  40 & GO:0007030 & Golgi organization &  86 &  35 & 10.98 & 8.3e-11 & BP \\ 
  41 & GO:0042793 & transcription from plastid promoter &  68 &  31 & 8.68 & 1.1e-10 & BP \\ 
  42 & GO:0009407 & toxin catabolic process &  61 &  28 & 7.79 & 2.2e-10 & BP \\ 
  43 & GO:0009657 & plastid organization & 361 & 162 & 46.08 & 5.1e-10 & BP \\ 
  44 & GO:0002181 & cytoplasmic translation &  31 &  20 & 3.96 & 6.5e-10 & BP \\ 
  45 & GO:0006833 & water transport &  62 &  27 & 7.91 & 2.0e-09 & BP \\ 
  46 & GO:0000398 & mRNA splicing, via spliceosome & 138 &  58 & 17.61 & 2.0e-09 & BP \\ 
  47 & GO:0042742 & defense response to bacterium & 113 &  40 & 14.42 & 5.8e-09 & BP \\ 
  48 & GO:0046686 & response to cadmium ion & 103 &  36 & 13.15 & 6.3e-09 & BP \\ 
  49 & GO:0006972 & hyperosmotic response &  62 &  24 & 7.91 & 6.8e-09 & BP \\ 
  50 & GO:0000413 & protein peptidyl-prolyl isomerization &  61 &  26 & 7.79 & 6.8e-09 & BP \\ 
  51 & GO:0006623 & protein targeting to vacuole &  65 &  27 & 8.30 & 6.9e-09 & BP \\ 
  52 & GO:0006886 & intracellular protein transport & 596 & 197 & 76.08 & 1.0e-08 & BP \\ 
  53 & GO:0030003 & cellular cation homeostasis & 139 &  39 & 17.74 & 1.5e-08 & BP \\ 
  54 & GO:0018298 & protein-chromophore linkage &  48 &  22 & 6.13 & 1.9e-08 & BP \\ 
  55 & GO:0010264 & myo-inositol hexakisphosphate biosynthet... &  35 &  18 & 4.47 & 4.0e-08 & BP \\ 
  56 & GO:0009644 & response to high light intensity &  77 &  31 & 9.83 & 4.4e-08 & BP \\ 
  57 & GO:0010103 & stomatal complex morphogenesis &  77 &  27 & 9.83 & 4.8e-08 & BP \\ 
  58 & GO:0009073 & aromatic amino acid family biosynthetic ... & 136 &  39 & 17.36 & 5.0e-08 & BP \\ 
  59 & GO:0006766 & vitamin metabolic process & 319 &  62 & 40.72 & 5.9e-08 & BP \\ 
  60 & GO:0045036 & protein targeting to chloroplast &  45 &  25 & 5.74 & 6.1e-08 & BP \\ 
  61 & GO:0009658 & chloroplast organization & 211 &  89 & 26.93 & 6.6e-08 & BP \\ 
  62 & GO:0006655 & phosphatidylglycerol biosynthetic proces... &  55 &  23 & 7.02 & 7.7e-08 & BP \\ 
  63 & GO:0006546 & glycine catabolic process &  31 &  16 & 3.96 & 7.8e-08 & BP \\ 
  64 & GO:0009768 & photosynthesis, light harvesting in phot... &  15 &  11 & 1.91 & 1.2e-07 & BP \\ 
  65 & GO:0009106 & lipoate metabolic process &  25 &  14 & 3.19 & 3.4e-07 & BP \\ 
  66 & GO:0015996 & chlorophyll catabolic process &  42 &  19 & 5.36 & 5.3e-07 & BP \\ 
  67 & GO:0009266 & response to temperature stimulus & 298 &  85 & 38.04 & 5.6e-07 & BP \\ 
  68 & GO:0006120 & mitochondrial electron transport, NADH t... &  69 &  25 & 8.81 & 5.8e-07 & BP \\ 
  69 & GO:0070838 & divalent metal ion transport & 152 &  44 & 19.40 & 6.0e-07 & BP \\ 
  70 & GO:0006470 & protein dephosphorylation & 238 &  80 & 30.38 & 6.2e-07 & BP \\ 
  71 & GO:0031408 & oxylipin biosynthetic process &  23 &  13 & 2.94 & 7.6e-07 & BP \\ 
  72 & GO:0032880 & regulation of protein localization &  18 &  12 & 2.30 & 8.5e-07 & BP \\ 
  73 & GO:0007033 & vacuole organization &  55 &  23 & 7.02 & 1.1e-06 & BP \\ 
  74 & GO:0009416 & response to light stimulus & 474 & 146 & 60.50 & 1.2e-06 & BP \\ 
  75 & GO:0048481 & plant ovule development &  67 &  24 & 8.55 & 1.2e-06 & BP \\ 
  76 & GO:0034660 & ncRNA metabolic process & 687 & 159 & 87.69 & 2.1e-06 & BP \\ 
  77 & GO:0016226 & iron-sulfur cluster assembly & 108 &  33 & 13.79 & 2.3e-06 & BP \\ 
  78 & GO:0000245 & spliceosomal complex assembly &  24 &  14 & 3.06 & 2.5e-06 & BP \\ 
  79 & GO:0032544 & plastid translation &  10 &   8 & 1.28 & 2.5e-06 & BP \\ 
  80 & GO:0000165 & MAPK cascade &  80 &  24 & 10.21 & 2.6e-06 & BP \\ 
  81 & GO:0055114 & oxidation-reduction process & 2564 & 483 & 327.28 & 3.2e-06 & BP \\ 
  82 & GO:0052541 & plant-type cell wall cellulose metabolic... &   8 &   7 & 1.02 & 3.9e-06 & BP \\ 
  83 & GO:0010304 & PSII associated light-harvesting complex... &  19 &  11 & 2.43 & 4.1e-06 & BP \\ 
  84 & GO:0016192 & vesicle-mediated transport & 416 & 112 & 53.10 & 4.2e-06 & BP \\ 
  85 & GO:0070534 & protein K63-linked ubiquitination &   6 &   6 & 0.77 & 4.3e-06 & BP \\ 
  86 & GO:0006448 & regulation of translational elongation &  71 &  23 & 9.06 & 4.3e-06 & BP \\ 
  87 & GO:0006744 & ubiquinone biosynthetic process & 113 &  33 & 14.42 & 4.4e-06 & BP \\ 
  88 & GO:0010200 & response to chitin &  58 &  21 & 7.40 & 4.6e-06 & BP \\ 
  89 & GO:0009793 & embryo development ending in seed dorman... & 239 &  54 & 30.51 & 5.2e-06 & BP \\ 
  90 & GO:0015986 & ATP synthesis coupled proton transport &  46 &  18 & 5.87 & 6.2e-06 & BP \\ 
  91 & GO:0010310 & regulation of hydrogen peroxide metaboli... &  34 &  15 & 4.34 & 6.4e-06 & BP \\ 
  92 & GO:0006007 & glucose catabolic process &  42 &  17 & 5.36 & 6.5e-06 & BP \\ 
  93 & GO:0009744 & response to sucrose &  70 &  24 & 8.94 & 8.2e-06 & BP \\ 
  94 & GO:0006397 & mRNA processing & 210 &  81 & 26.81 & 9.2e-06 & BP \\ 
  95 & GO:0015991 & ATP hydrolysis coupled proton transport &  43 &  17 & 5.49 & 9.5e-06 & BP \\ 
  96 & GO:0009408 & response to heat & 104 &  31 & 13.27 & 9.9e-06 & BP \\ 
  97 & GO:0006012 & galactose metabolic process &  70 &  23 & 8.94 & 1.1e-05 & BP \\ 
  98 & GO:0009697 & salicylic acid biosynthetic process &  44 &  17 & 5.62 & 1.4e-05 & BP \\ 
  99 & GO:0000387 & spliceosomal snRNP assembly &  14 &  10 & 1.79 & 1.6e-05 & BP \\ 
  100 & GO:0006913 & nucleocytoplasmic transport & 180 &  55 & 22.98 & 1.6e-05 & BP \\ 
  101 & GO:0003735 & structural constituent of ribosome & 432 & 171 & 52.01 & $<$ 1e-30 & MF \\ 
  102 & GO:0005515 & protein binding & 3192 & 639 & 384.32 & 1.8e-22 & MF \\ 
  103 & GO:0005525 & GTP binding & 287 &  93 & 34.56 & 6.1e-20 & MF \\ 
  104 & GO:0051082 & unfolded protein binding &  91 &  44 & 10.96 & 1.6e-17 & MF \\ 
  105 & GO:0016168 & chlorophyll binding &  40 &  24 & 4.82 & 7.2e-13 & MF \\ 
  106 & GO:0003924 & GTPase activity & 176 &  57 & 21.19 & 9.0e-13 & MF \\ 
  107 & GO:0003743 & translation initiation factor activity & 105 &  40 & 12.64 & 7.2e-12 & MF \\ 
  108 & GO:0000166 & nucleotide binding & 4725 & 723 & 568.90 & 2.9e-10 & MF \\ 
  109 & GO:0061630 & ubiquitin protein ligase activity &  72 &  29 & 8.67 & 1.2e-09 & MF \\ 
  110 & GO:0003723 & RNA binding & 1026 & 258 & 123.53 & 1.3e-09 & MF \\ 
  111 & GO:0005509 & calcium ion binding & 320 &  77 & 38.53 & 1.6e-09 & MF \\ 
  112 & GO:0003755 & peptidyl-prolyl cis-trans isomerase acti... &  61 &  26 & 7.34 & 2.0e-09 & MF \\ 
  113 & GO:0031625 & ubiquitin protein ligase binding &  38 &  19 & 4.58 & 1.2e-08 & MF \\ 
  114 & GO:0019904 & protein domain specific binding &  15 &  11 & 1.81 & 6.5e-08 & MF \\ 
  115 & GO:0031072 & heat shock protein binding &  36 &  18 & 4.33 & 1.2e-07 & MF \\ 
  116 & GO:0016157 & sucrose synthase activity &  16 &  11 & 1.93 & 1.9e-07 & MF \\ 
  117 & GO:0008137 & NADH dehydrogenase (ubiquinone) activity &  75 &  26 & 9.03 & 3.0e-07 & MF \\ 
  118 & GO:0003746 & translation elongation factor activity &  54 &  21 & 6.50 & 4.5e-07 & MF \\ 
  119 & GO:0004298 & threonine-type endopeptidase activity &  18 &  11 & 2.17 & 1.1e-06 & MF \\ 
  120 & GO:0005528 & FK506 binding &  19 &  11 & 2.29 & 2.3e-06 & MF \\ 
  121 & GO:0031409 & pigment binding &  14 &  10 & 1.69 & 2.4e-06 & MF \\ 
  122 & GO:0004722 & protein serine/threonine phosphatase act... &  83 &  25 & 9.99 & 7.1e-06 & MF \\ 
  123 & GO:0046524 & sucrose-phosphate synthase activity &   9 &   7 & 1.08 & 1.1e-05 & MF \\ 
  124 & GO:0008139 & nuclear localization sequence binding &  18 &  10 & 2.17 & 1.1e-05 & MF \\ 
  125 & GO:0043130 & ubiquitin binding &  12 &   8 & 1.44 & 1.4e-05 & MF \\ 
  126 & GO:0051537 & 2 iron, 2 sulfur cluster binding &  51 &  18 & 6.14 & 1.5e-05 & MF \\ 
  127 & GO:0046961 & proton-transporting ATPase activity, rot... &  34 &  14 & 4.09 & 1.7e-05 & MF \\ 
  128 & GO:0005198 & structural molecule activity & 530 & 200 & 63.81 & 2.0e-05 & MF \\ 
  129 & GO:0019843 & rRNA binding &  91 &  32 & 10.96 & 4.7e-05 & MF \\ 
  130 & GO:0004176 & ATP-dependent peptidase activity &  29 &  12 & 3.49 & 6.5e-05 & MF \\ 
  131 & GO:0046872 & metal ion binding & 5795 & 662 & 697.73 & 9.2e-05 & MF \\ 
  132 & GO:0032440 & 2-alkenal reductase [NAD(P)] activity &  63 &  19 & 7.59 & 0.00010 & MF \\ 
  133 & GO:0001055 & RNA polymerase II activity &  12 &   7 & 1.44 & 0.00017 & MF \\ 
  134 & GO:0045548 & phenylalanine ammonia-lyase activity &   9 &   6 & 1.08 & 0.00018 & MF \\ 
  135 & GO:0008565 & protein transporter activity & 104 &  33 & 12.52 & 0.00020 & MF \\ 
  136 & GO:0015078 & hydrogen ion transmembrane transporter a... & 211 &  56 & 25.40 & 0.00022 & MF \\ 
  137 & GO:0042803 & protein homodimerization activity &  62 &  18 & 7.46 & 0.00026 & MF \\ 
  138 & GO:0016874 & ligase activity & 726 & 107 & 87.41 & 0.00028 & MF \\ 
  139 & GO:0043022 & ribosome binding &  33 &  12 & 3.97 & 0.00028 & MF \\ 
  140 & GO:0004004 & ATP-dependent RNA helicase activity &  96 &  24 & 11.56 & 0.00035 & MF \\ 
  141 & GO:0030955 & potassium ion binding &  17 &   8 & 2.05 & 0.00039 & MF \\ 
  142 & GO:0004743 & pyruvate kinase activity &  17 &   8 & 2.05 & 0.00039 & MF \\ 
  143 & GO:0004784 & superoxide dismutase activity &  10 &   6 & 1.20 & 0.00041 & MF \\ 
  144 & GO:0042802 & identical protein binding & 132 &  37 & 15.89 & 0.00044 & MF \\ 
  145 & GO:0004842 & ubiquitin-protein transferase activity & 226 &  62 & 27.21 & 0.00060 & MF \\ 
  146 & GO:0045156 & electron transporter, transferring elect... &  18 &   8 & 2.17 & 0.00063 & MF \\ 
  147 & GO:0017025 & TBP-class protein binding &  11 &   6 & 1.32 & 0.00082 & MF \\ 
  148 & GO:0008553 & hydrogen-exporting ATPase activity, phos... &  23 &   9 & 2.77 & 0.00088 & MF \\ 
  149 & GO:0004721 & phosphoprotein phosphatase activity & 155 &  41 & 18.66 & 0.00090 & MF \\ 
  150 & GO:0016671 & oxidoreductase activity, acting on a sul... &  54 &  17 & 6.50 & 0.00092 & MF \\ 
  151 & GO:0005200 & structural constituent of cytoskeleton &  37 &  12 & 4.45 & 0.00092 & MF \\ 
  152 & GO:0003729 & mRNA binding &  45 &  14 & 5.42 & 0.00094 & MF \\ 
  153 & GO:0004427 & inorganic diphosphatase activity &   8 &   5 & 0.96 & 0.00103 & MF \\ 
  154 & GO:0016688 & L-ascorbate peroxidase activity &   8 &   5 & 0.96 & 0.00103 & MF \\ 
  155 & GO:0070181 & small ribosomal subunit rRNA binding &   8 &   5 & 0.96 & 0.00103 & MF \\ 
  156 & GO:0046933 & proton-transporting ATP synthase activit... &  33 &  11 & 3.97 & 0.00116 & MF \\ 
  157 & GO:0004222 & metalloendopeptidase activity &  81 &  20 & 9.75 & 0.00123 & MF \\ 
  158 & GO:0008047 & enzyme activator activity &  66 &  20 & 7.95 & 0.00145 & MF \\ 
  159 & GO:0043424 & protein histidine kinase binding &  12 &   6 & 1.44 & 0.00147 & MF \\ 
  160 & GO:0005524 & ATP binding & 3193 & 437 & 384.44 & 0.00149 & MF \\ 
  161 & GO:0015035 & protein disulfide oxidoreductase activit... &  77 &  19 & 9.27 & 0.00162 & MF \\ 
  162 & GO:0035798 & 2-alkenal reductase (NADP+) activity &   3 &   3 & 0.36 & 0.00174 & MF \\ 
  163 & GO:0035671 & enone reductase activity &   3 &   3 & 0.36 & 0.00174 & MF \\ 
  164 & GO:0005506 & iron ion binding & 310 &  56 & 37.32 & 0.00195 & MF \\ 
  165 & GO:0000822 & inositol hexakisphosphate binding &   9 &   5 & 1.08 & 0.00209 & MF \\ 
  166 & GO:0004707 & MAP kinase activity &  17 &   7 & 2.05 & 0.00236 & MF \\ 
  167 & GO:0042132 & fructose 1,6-bisphosphate 1-phosphatase ... &   6 &   4 & 0.72 & 0.00257 & MF \\ 
  168 & GO:0050897 & cobalt ion binding &  47 &  13 & 5.66 & 0.00292 & MF \\ 
  169 & GO:0008536 & Ran GTPase binding &  32 &  10 & 3.85 & 0.00331 & MF \\ 
  170 & GO:0015450 & P-P-bond-hydrolysis-driven protein trans... &  13 &   7 & 1.57 & 0.00376 & MF \\ 
  171 & GO:0009882 & blue light photoreceptor activity &  10 &   5 & 1.20 & 0.00376 & MF \\ 
  172 & GO:0000287 & magnesium ion binding & 192 &  36 & 23.12 & 0.00455 & MF \\ 
  173 & GO:0035091 & phosphatidylinositol binding &  33 &  12 & 3.97 & 0.00489 & MF \\ 
  174 & GO:0004602 & glutathione peroxidase activity &   7 &   4 & 0.84 & 0.00543 & MF \\ 
  175 & GO:0080130 & L-phenylalanine:2-oxoglutarate aminotran... &   7 &   4 & 0.84 & 0.00543 & MF \\ 
  176 & GO:0004739 & pyruvate dehydrogenase (acetyl-transferr... &   7 &   4 & 0.84 & 0.00543 & MF \\ 
  177 & GO:0030060 & L-malate dehydrogenase activity &   7 &   4 & 0.84 & 0.00543 & MF \\ 
  178 & GO:0015266 & protein channel activity &  11 &   5 & 1.32 & 0.00622 & MF \\ 
  179 & GO:0019888 & protein phosphatase regulator activity &  23 &   6 & 2.77 & 0.00622 & MF \\ 
  180 & GO:0043021 & ribonucleoprotein complex binding &  45 &  19 & 5.42 & 0.00628 & MF \\ 
  181 & GO:0000386 & second spliceosomal transesterification ... &   4 &   3 & 0.48 & 0.00635 & MF \\ 
  182 & GO:0008266 & poly(U) RNA binding &   4 &   3 & 0.48 & 0.00635 & MF \\ 
  183 & GO:0004128 & cytochrome-b5 reductase activity, acting... &   4 &   3 & 0.48 & 0.00635 & MF \\ 
  184 & GO:0016710 & trans-cinnamate 4-monooxygenase activity &   4 &   3 & 0.48 & 0.00635 & MF \\ 
  185 & GO:0009055 & electron carrier activity & 265 &  57 & 31.91 & 0.00651 & MF \\ 
  186 & GO:0051087 & chaperone binding &  25 &   8 & 3.01 & 0.00714 & MF \\ 
  187 & GO:0005507 & copper ion binding & 157 &  31 & 18.90 & 0.00718 & MF \\ 
  188 & GO:0009931 & calcium-dependent protein serine/threoni... &  52 &  13 & 6.26 & 0.00744 & MF \\ 
  189 & GO:0030276 & clathrin binding &  14 &   6 & 1.69 & 0.00961 & MF \\ 
  190 & GO:0004747 & ribokinase activity &  12 &   5 & 1.44 & 0.00962 & MF \\ 
  191 & GO:0004069 & L-aspartate:2-oxoglutarate aminotransfer... &   8 &   4 & 0.96 & 0.00983 & MF \\ 
  192 & GO:0036402 & proteasome-activating ATPase activity &   8 &   4 & 0.96 & 0.00983 & MF \\ 
  193 & GO:0004843 & thiol-dependent ubiquitin-specific prote... &   8 &   4 & 0.96 & 0.00983 & MF \\ 
  194 & GO:0046910 & pectinesterase inhibitor activity &  22 &   7 & 2.65 & 0.01204 & MF \\ 
  195 & GO:2001070 & starch binding &  13 &   5 & 1.57 & 0.01410 & MF \\ 
  196 & GO:0005388 & calcium-transporting ATPase activity &  13 &   5 & 1.57 & 0.01410 & MF \\ 
  197 & GO:0019903 & protein phosphatase binding &   9 &   4 & 1.08 & 0.01444 & MF \\ 
  198 & GO:0004449 & isocitrate dehydrogenase (NAD+) activity &   5 &   3 & 0.60 & 0.01444 & MF \\ 
  199 & GO:0070180 & large ribosomal subunit rRNA binding &   5 &   3 & 0.60 & 0.01444 & MF \\ 
  200 & GO:0004775 & succinate-CoA ligase (ADP-forming) activ... &   5 &   3 & 0.60 & 0.01444 & MF \\ 
  201 & GO:0009941 & chloroplast envelope & 425 & 193 & 66.13 & $<$ 1e-30 & CC \\ 
  202 & GO:0009535 & chloroplast thylakoid membrane & 227 & 130 & 35.32 & $<$ 1e-30 & CC \\ 
  203 & GO:0009570 & chloroplast stroma & 491 & 213 & 76.40 & $<$ 1e-30 & CC \\ 
  204 & GO:0005829 & cytosol & 1091 & 381 & 169.75 & $<$ 1e-30 & CC \\ 
  205 & GO:0009507 & chloroplast & 1878 & 660 & 292.20 & 1.0e-25 & CC \\ 
  206 & GO:0009534 & chloroplast thylakoid & 310 & 182 & 48.23 & 3.1e-24 & CC \\ 
  207 & GO:0005840 & ribosome & 636 & 252 & 98.96 & 5.6e-23 & CC \\ 
  208 & GO:0022625 & cytosolic large ribosomal subunit &  96 &  56 & 14.94 & 9.1e-22 & CC \\ 
  209 & GO:0005737 & cytoplasm & 8378 & 1920 & 1303.55 & 8.6e-21 & CC \\ 
  210 & GO:0048046 & apoplast & 174 &  70 & 27.07 & 3.1e-15 & CC \\ 
  211 & GO:0005622 & intracellular & 12244 & 2408 & 1905.06 & 9.1e-14 & CC \\ 
  212 & GO:0022627 & cytosolic small ribosomal subunit &  37 &  24 & 5.76 & 1.7e-11 & CC \\ 
  213 & GO:0010287 & plastoglobule &  24 &  18 & 3.73 & 1.4e-10 & CC \\ 
  214 & GO:0031977 & thylakoid lumen &  45 &  25 & 7.00 & 1.8e-10 & CC \\ 
  215 & GO:0010319 & stromule &  25 &  18 & 3.89 & 4.3e-10 & CC \\ 
  216 & GO:0009579 & thylakoid & 490 & 234 & 76.24 & 2.3e-09 & CC \\ 
  217 & GO:0005774 & vacuolar membrane & 239 &  73 & 37.19 & 7.7e-09 & CC \\ 
  218 & GO:0005794 & Golgi apparatus & 526 & 122 & 81.84 & 9.6e-09 & CC \\ 
  219 & GO:0005747 & mitochondrial respiratory chain complex ... &  37 &  20 & 5.76 & 7.0e-08 & CC \\ 
  220 & GO:0005886 & plasma membrane & 1336 & 270 & 207.87 & 6.6e-07 & CC \\ 
  221 & GO:0005783 & endoplasmic reticulum & 404 & 114 & 62.86 & 9.3e-07 & CC \\ 
  222 & GO:0009506 & plasmodesma & 338 &  86 & 52.59 & 1.5e-06 & CC \\ 
  223 & GO:0005773 & vacuole & 522 & 143 & 81.22 & 6.0e-06 & CC \\ 
  224 & GO:0071013 & catalytic step 2 spliceosome &  37 &  17 & 5.76 & 1.2e-05 & CC \\ 
  225 & GO:0005686 & U2 snRNP &  18 &  11 & 2.80 & 1.4e-05 & CC \\ 
  226 & GO:0009840 & chloroplastic endopeptidase Clp complex &   6 &   6 & 0.93 & 1.4e-05 & CC \\ 
  227 & GO:0010598 & NAD(P)H dehydrogenase complex (plastoqui... &   8 &   7 & 1.24 & 1.5e-05 & CC \\ 
  228 & GO:0009522 & photosystem I &  54 &  26 & 8.40 & 1.6e-05 & CC \\ 
  229 & GO:0030126 & COPI vesicle coat &  11 &   8 & 1.71 & 3.6e-05 & CC \\ 
  230 & GO:0005682 & U5 snRNP &  12 &   8 & 1.87 & 9.3e-05 & CC \\ 
  231 & GO:0071011 & precatalytic spliceosome &  21 &  11 & 3.27 & 9.8e-05 & CC \\ 
  232 & GO:0033290 & eukaryotic 48S preinitiation complex &  15 &   9 & 2.33 & 0.00011 & CC \\ 
  233 & GO:0030529 & intracellular ribonucleoprotein complex & 918 & 341 & 142.83 & 0.00013 & CC \\ 
  234 & GO:0070469 & respiratory chain & 123 &  49 & 19.14 & 0.00018 & CC \\ 
  235 & GO:0019013 & viral nucleocapsid &  55 &  19 & 8.56 & 0.00041 & CC \\ 
  236 & GO:0005689 & U12-type spliceosomal complex &  14 &   8 & 2.18 & 0.00042 & CC \\ 
  237 & GO:0008287 & protein serine/threonine phosphatase com... &  91 &  28 & 14.16 & 0.00060 & CC \\ 
  238 & GO:0009523 & photosystem II & 129 &  41 & 20.07 & 0.00064 & CC \\ 
  239 & GO:0005665 & DNA-directed RNA polymerase II, core com... &  18 &   9 & 2.80 & 0.00067 & CC \\ 
  240 & GO:0071004 & U2-type prespliceosome &  19 &   9 & 2.96 & 0.00110 & CC \\ 
  241 & GO:0046540 & U4/U6 x U5 tri-snRNP complex &  16 &   8 & 2.49 & 0.00135 & CC \\ 
  242 & GO:0016282 & eukaryotic 43S preinitiation complex &  13 &   7 & 2.02 & 0.00158 & CC \\ 
  243 & GO:0005839 & proteasome core complex &  18 &  11 & 2.80 & 0.00167 & CC \\ 
  244 & GO:0005685 & U1 snRNP &  20 &   9 & 3.11 & 0.00172 & CC \\ 
  245 & GO:0097526 & spliceosomal tri-snRNP complex &  21 &  12 & 3.27 & 0.00254 & CC \\ 
  246 & GO:0009538 & photosystem I reaction center &   5 &   4 & 0.78 & 0.00256 & CC \\ 
  247 & GO:0005853 & eukaryotic translation elongation factor... &   5 &   4 & 0.78 & 0.00256 & CC \\ 
  248 & GO:0005681 & spliceosomal complex &  88 &  35 & 13.69 & 0.00271 & CC \\ 
  249 & GO:0005852 & eukaryotic translation initiation factor... &  14 &   7 & 2.18 & 0.00273 & CC \\ 
  250 & GO:0000325 & plant-type vacuole &  57 &  12 & 8.87 & 0.00274 & CC \\ 
  251 & GO:0009654 & photosystem II oxygen evolving complex &  50 &  16 & 7.78 & 0.00282 & CC \\ 
  252 & GO:0005623 & cell & 13500 & 2557 & 2100.49 & 0.00288 & CC \\ 
  253 & GO:0005762 & mitochondrial large ribosomal subunit &  29 &  11 & 4.51 & 0.00288 & CC \\ 
  254 & GO:0000243 & commitment complex &  11 &   6 & 1.71 & 0.00321 & CC \\ 
  255 & GO:0019773 & proteasome core complex, alpha-subunit c... &   8 &   5 & 1.24 & 0.00337 & CC \\ 
  256 & GO:0045254 & pyruvate dehydrogenase complex &   8 &   5 & 1.24 & 0.00337 & CC \\ 
  257 & GO:0005654 & nucleoplasm & 257 &  52 & 39.99 & 0.00338 & CC \\ 
  258 & GO:0005789 & endoplasmic reticulum membrane & 163 &  43 & 25.36 & 0.00358 & CC \\ 
  259 & GO:0031361 & integral component of thylakoid membrane &   3 &   3 & 0.47 & 0.00376 & CC \\ 
  260 & GO:0000311 & plastid large ribosomal subunit &   3 &   3 & 0.47 & 0.00376 & CC \\ 
  261 & GO:0034715 & pICln-Sm protein complex &   3 &   3 & 0.47 & 0.00376 & CC \\ 
  262 & GO:0005730 & nucleolus & 254 &  61 & 39.52 & 0.00498 & CC \\ 
  263 & GO:0030687 & preribosome, large subunit precursor &   9 &   5 & 1.40 & 0.00660 & CC \\ 
  264 & GO:0015030 & Cajal body &   6 &   4 & 0.93 & 0.00673 & CC \\ 
  265 & GO:0031305 & integral component of mitochondrial inne... &   6 &   4 & 0.93 & 0.00673 & CC \\ 
  266 & GO:0010494 & cytoplasmic stress granule &   6 &   4 & 0.93 & 0.00673 & CC \\ 
  267 & GO:0009706 & chloroplast inner membrane &  30 &  11 & 4.67 & 0.00953 & CC \\ 
  268 & GO:0009508 & plastid chromosome &  10 &   5 & 1.56 & 0.01153 & CC \\ 
  269 & GO:0005618 & cell wall & 295 &  58 & 45.90 & 0.01308 & CC \\ 
  270 & GO:0000276 & mitochondrial proton-transporting ATP sy... &   4 &   3 & 0.62 & 0.01330 & CC \\ 
  271 & GO:0005771 & multivesicular body &   4 &   3 & 0.62 & 0.01330 & CC \\ 
  272 & GO:0009512 & cytochrome b6f complex &   4 &   3 & 0.62 & 0.01330 & CC \\ 
  273 & GO:0008540 & proteasome regulatory particle, base sub... &  14 &   6 & 2.18 & 0.01372 & CC \\ 
  274 & GO:0005751 & mitochondrial respiratory chain complex ... &   7 &   4 & 1.09 & 0.01378 & CC \\ 
  275 & GO:0000220 & vacuolar proton-transporting V-type ATPa... &   7 &   4 & 1.09 & 0.01378 & CC \\ 
  276 & GO:0000974 & Prp19 complex &   7 &   4 & 1.09 & 0.01378 & CC \\ 
  277 & GO:0009573 & chloroplast ribulose bisphosphate carbox... &   7 &   4 & 1.09 & 0.01378 & CC \\ 
  278 & GO:0009504 & cell plate &   7 &   4 & 1.09 & 0.01378 & CC \\ 
  279 & GO:0005795 & Golgi stack &  49 &   6 & 7.62 & 0.01386 & CC \\ 
  280 & GO:0031965 & nuclear membrane &  22 &   8 & 3.42 & 0.01407 & CC \\ 
  281 & GO:0016529 & sarcoplasmic reticulum &  11 &   5 & 1.71 & 0.01846 & CC \\ 
  282 & GO:0000502 & proteasome complex &  54 &  26 & 8.40 & 0.01924 & CC \\ 
  283 & GO:0009332 & glutamate-tRNA ligase complex &   2 &   2 & 0.31 & 0.02420 & CC \\ 
  284 & GO:0031372 & UBC13-MMS2 complex &   2 &   2 & 0.31 & 0.02420 & CC \\ 
  285 & GO:0043614 & multi-eIF complex &   2 &   2 & 0.31 & 0.02420 & CC \\ 
  286 & GO:0080085 & signal recognition particle, chloroplast... &   2 &   2 & 0.31 & 0.02420 & CC \\ 
  287 & GO:0005850 & eukaryotic translation initiation factor... &   2 &   2 & 0.31 & 0.02420 & CC \\ 
  288 & GO:0016607 & nuclear speck &   8 &   4 & 1.24 & 0.02420 & CC \\ 
  289 & GO:0031595 & nuclear proteasome complex &   8 &   4 & 1.24 & 0.02420 & CC \\ 
  290 & GO:0005929 & cilium &  24 &   5 & 3.73 & 0.02425 & CC \\ 
  291 & GO:0019898 & extrinsic component of membrane &  96 &  22 & 14.94 & 0.02445 & CC \\ 
  292 & GO:0034399 & nuclear periphery &  21 &   8 & 3.27 & 0.02630 & CC \\ 
  293 & GO:0022626 & cytosolic ribosome & 158 &  88 & 24.58 & 0.02775 & CC \\ 
  294 & GO:0071010 & prespliceosome &  24 &  12 & 3.73 & 0.02924 & CC \\ 
  295 & GO:0005753 & mitochondrial proton-transporting ATP sy... &  14 &   9 & 2.18 & 0.02928 & CC \\ 
  296 & GO:0045275 & respiratory chain complex III &  12 &   6 & 1.87 & 0.02935 & CC \\ 
  297 & GO:0005845 & mRNA cap binding complex &   5 &   3 & 0.78 & 0.02940 & CC \\ 
  298 & GO:0042709 & succinate-CoA ligase complex &   5 &   3 & 0.78 & 0.02940 & CC \\ 
  299 & GO:0005948 & acetolactate synthase complex &   5 &   3 & 0.78 & 0.02940 & CC \\ 
  300 & GO:0000275 & mitochondrial proton-transporting ATP sy... &   5 &   3 & 0.78 & 0.02940 & CC \\ 
  \hline
\end{longtable}

\end{document}